app/models/reservation.rb

```diff
@@ -78,6 +78,6 @@ def bigger(time_a, time_b)
   end
 
   def check_cancellation
-    MakersMailer.cancellation_email(self).deliver if cancelled?
+    MakersMailer.cancellation_email(self).deliver if rejected?
```

Comment:
From this I understand that users will only get cancellation e-mails when admins reject their reservations and NOT when the user cancels their own reservation? If so, then disregard my comment about changing the e-mail contents back to "cancelar" instead of "rechazar"

Reply:
Yes, the email is only sent when it is rejected.